functions/tech-support

```diff
@@ -314,257 +331,299 @@ if cli-shell-api existsActive protocols bgp; then
+    #header IP Access Lists
+    #show ip access-lists
+    header IP Community List
+    show ip community-list 
+  else
+    echo "Policy is not configured"
+  fi
+  
+  header Traffic Policy 
+  if cli-shell-api existsActive traffic-policy; then
+    header Current Traffic Policies
+    show queueing
+  else
+    echo "Traffic-Policy is not configured"
+  fi
 
-header RIP
-if cli-shell-api existsActive protocols rip; then
-  header IP RIP
-  show ip rip 
-  header RIP Status
-  show ip rip status
-  header RIP Debugging Information
-  show monitoring protocols rip 
-else
-  echo "RIP is not configured"
-fi
+  header RIP
+  if cli-shell-api existsActive protocols rip; then
+    header IP RIP
+    show ip rip 
+    header RIP Status
+    show ip rip status
+    header RIP Debugging Information
+    show monitoring protocols rip 
+  else
+    echo "RIP is not configured"
+  fi
 
-header RIPNG 
-if cli-shell-api existsActive protocols ripng; then
-  header RIPNG Debugging Information
-  show monitoring protocols ripng
-else
-  echo "RIPNG is not configured"
-fi
+  header RIPNG 
+  if cli-shell-api existsActive protocols ripng; then
+    header RIPNG Debugging Information
+    show monitoring protocols ripng
+  else
+    echo "RIPNG is not configured"
+  fi
 
-header VPN-L2TP 
-if cli-shell-api existsActive vpn l2tp; then
-  header VPN ike secrets
-  show vpn ike secrets
-  header VPN rsa-keys
-  show vpn ike rsa-keys  
-  header VPN ike sa
-  show vpn ike sa
-  header VPN ike Status
-  show vpn ike status
-  header VPN Remote-Access
-  show vpn remote-access
-
-  header VPN Debug Detail
-  show vpn debug detail
-else
-  echo "VPN L2TP is not configured"
-fi
+  header VPN-L2TP 
+  if cli-shell-api existsActive vpn l2tp; then
+    header VPN ike secrets
+    show vpn ike secrets
+    header VPN rsa-keys
+    show vpn ike rsa-keys  
+    header VPN ike sa
+    show vpn ike sa
+    header VPN ike Status
+    show vpn ike status
+    header VPN Remote-Access
+    show vpn remote-access
+
+    header VPN Debug Detail
+    show vpn debug detail
+  else
+    echo "VPN L2TP is not configured"
+  fi
 
-header VPN-PPTP 
-if cli-shell-api existsActive vpn pptp; then
-  header VPN Remote-Access
-  show vpn remote-access
-else
-  echo "VPN PPTP is not configured"
-fi
+  header VPN-PPTP 
+  if cli-shell-api existsActive vpn pptp; then
+    header VPN Remote-Access
+    show vpn remote-access
+  else
+    echo "VPN PPTP is not configured"
+  fi
 
-header VRRP
-# XXX: not checking if configured, we'd have to walk all VIFs
-show vrrp detail
-
-header WAN LOAD BALANCING
-if cli-shell-api existsActive load-balancing wan; then
-  header Wan Load Balance
-  show wan-load-balance 
-  header Wan Load Balance Status 
-  show wan-load-balance status
-  header Wan Load Balance Connection
-  show wan-load-balance connection
-else
-  echo "Wan Load Balance is not configured"
-fi
+  header VRRP
+  # XXX: not checking if configured, we'd have to walk all VIFs
```

Comment:
A long solved problem: VRRP is in "high-availability vrrp".

Reply:
Maybe replace is with show vrrp detail command?

Reply:
Oh right, removing a comment that describes a problem solves the problem.
You can see that before gathering outputs from every component, the script tries to find out if it's configured to begin with. Like in `if cli-shell-api existsActive protocols ospf;`. 

VRRP doesn't fit that pattern. The comment vaguely explains why: it was hard to do. I clearly say why it's no longer the case: VRRP is a single top level path "high-availability vrrp".

Reply:
I have added checking for VRRP  .